### PR TITLE
[publus] add and update publus based websites

### LINKED ISSUE #2652: [Site Request] Jumpbookstore
**Name of the website**
Jumpbookstore

**Website urls (examples below)**
- Site: https://jumpbookstore.com
- Manga List: https://jumpbookstore.com/    for top sells (mangas and magazines) 
https://jumpbookstore.com/ext/jc.html  (Jump mangas)
https://jumpbookstore.com/ext/yjc.html  (YoungJump mangas)
https://jumpbookstore.com/ext/novel.html  (Novel part of the website)
- Manga example: https://jumpbookstore.com/item_list.html?keyword=%E9%80%B1%E5%88%8A%E5%B0%91%E5%B9%B4%E3%82%B8%E3%83%A3%E3%83%B3%E3%83%97+2020
- Chapter online viewer example: https://jumpbookstore.com/client_info/SHUEISHA/html/player/viewer.html?tw=2&lin=1&cid=SHSA_JP01WJ2020046D01_57

**Languages**
Japanese

**Website relationship**
One of the personal websites of Shueisha

**Additional details**
This website uses the pubplus tamplate

### PR DESCRIPTION
**Pull request by 27136773**

### CODE REVIEW

**Comment by newaccforhc** [comment]
Hmm.. I couldn't help but test this super duper alpha connector.. But, I can see so much blocky appearance in the downloaded images.. I'll share images and all on Discord DMs with you.. I also checked the website reader, but the images are smooth and don't have blocky appearance there.

**Comment by Ever4engel** [comment]
with this last update i see that blocky appearances in the downloaded images are just gone.

File: src/web/mjs/connectors/JumpBookStore.mjs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    async _getMangaFromURI(uri) {
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchDOM(request, 'head meta[property="og:title"]');
+        return new Manga(this, uri.pathname, data[0].content.trim());
+    }
+
+    async _getMangas() {
+        let mangaList = [];
+        for(let section of [ null ]) { // [ 'newb', 'newy', 'adultnew' ]
```
Comment (Sheepux) [inline src/web/mjs/connectors/JumpBookStore.mjs]: Is that supposed to evolve or is that some leftover ?

**Review by Sheepux — APPROVED** [review]

File: src/web/mjs/connectors/JumpBookStore.mjs (hunk 1 above)
Comment (27136773) [reply src/web/mjs/connectors/JumpBookStore.mjs]: Mh... seems like development artifacts
=> removed

Maybe this was supposed to be a parallel loop with a single cycle...

**Pull request merged**